### ISSUE TITLE
How to measure the memory footprint of the only model.

### ISSUE THREAD

**Issue by stigma0617** [facebookresearch/maskrcnn-benchmark#816]
## ❓ Questions and Help

Hi, 

My question is how to measure only the memory footprint a model requires excluding data, optimizer, etc.

The background of my question is that

I have analyzed the memory footprint of the Faster/Mask R-CNN models. 

For examples, 

The memory consumption the model requires is the sum of **model parameters** and **intermediate activation maps**, assuming exception for the memory whose is required for training process. 

I have searched the memory issuses in this repo and [the training  log](https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/maskrcnn_benchmark/engine/trainer.py#L105) is including model memory (weight+activation maps(forward-backward)) and data, training parameters. 

Let me know this is wrong. 

In conclusion, 

I want to check the memory footprint the model requires in training and inference, both.

training time: model parameters + intermediate activations (forward and backward features both)
inference time: model parameters + intermediate activations (only forward) 

Thanks for your advice in advance.
